File: tms_driver_app/lib/models/about_app_info.dart
// lib/models/about_app_info.dart

class AboutAppInfo {
  final String appNameKm;
  final String appNameEn;
  final String androidVersion;
  final String iosVersion;
  final String contactEmail;
  final String privacyPolicyUrlKm;
  final String privacyPolicyUrlEn;
  final String termsConditionsUrlKm;
  final String termsConditionsUrlEn;

  AboutAppInfo({
    required this.appNameKm,
    required this.appNameEn,
    required this.androidVersion,
    required this.iosVersion,
    required this.contactEmail,
    required this.privacyPolicyUrlKm,
    required this.privacyPolicyUrlEn,
    required this.termsConditionsUrlKm,
    required this.termsConditionsUrlEn,
  });

  factory AboutAppInfo.fromJson(Map<String, dynamic> json) {
    return AboutAppInfo(
      appNameKm: json['appNameKm'] ?? '',
      appNameEn: json['appNameEn'] ?? '',
      androidVersion: json['androidVersion'] ?? '',
      iosVersion: json['iosVersion'] ?? '',
      contactEmail: json['contactEmail'] ?? '',
      privacyPolicyUrlKm: json['privacyPolicyUrlKm'] ?? '',
      privacyPolicyUrlEn: json['privacyPolicyUrlEn'] ?? '',
      termsConditionsUrlKm: json['termsConditionsUrlKm'] ?? '',
      termsConditionsUrlEn: json['termsConditionsUrlEn'] ?? '',
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'appNameKm': appNameKm,
      'appNameEn': appNameEn,
      'androidVersion': androidVersion,
      'iosVersion': iosVersion,
      'contactEmail': contactEmail,
      'privacyPolicyUrlKm': privacyPolicyUrlKm,
      'privacyPolicyUrlEn': privacyPolicyUrlEn,
      'termsConditionsUrlKm': termsConditionsUrlKm,
      'termsConditionsUrlEn': termsConditionsUrlEn,
    };
  }
}

File: tms-core-api/src/main/java/com/svtrucking/logistics/model/AboutAppInfo.java
package com.svtrucking.logistics.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import java.time.LocalDateTime;
import lombok.Data;

@Entity
@Table(name = "about_app_info")
@Data
public class AboutAppInfo {
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private Long id;

  private String appNameKm;
  private String appNameEn;

  private String androidVersion;
  private String iosVersion;
  private String contactEmail;

  @Column(columnDefinition = "TEXT")
  private String privacyPolicyUrlKm;

  @Column(columnDefinition = "TEXT")
  private String privacyPolicyUrlEn;

  @Column(columnDefinition = "TEXT")
  private String termsConditionsUrlKm;

  @Column(columnDefinition = "TEXT")
  private String termsConditionsUrlEn;

  private LocalDateTime lastUpdated;
}
